geotrek/feedback/tests/test_suricate_sync.py

```diff
@@ -63,6 +70,11 @@ def build_response_patch(url, params=None, **kwargs):
             elif "GetAlerts" in url:
                 mock_response.content = mocked_json("suricate_alerts.json")
                 mock_response.status_code = 200
+            elif cause_JPG_error:
+                mock_response.status_code = 404
+            elif ".jpg" in url or ".png" in url or ".JPG" in url:
+                mock_response.content = mocked_image("theme-fauna.png")
+                mock_response.status_code = 200
```

Comment:
Do you think that adding a http `content_type` header on top of the jpg data would improve your tests ?